src/platforms/arm/stm32/fastpin_arm_stm32.h

```diff
@@ -93,7 +110,36 @@ _FL_IO(G,6);
 #endif
 
 // Actual pin definitions
-#if defined(SPARK) // Sparkfun STM32F103 based board
+#if defined(STM32F2XX) // Photon Particle
+
+// https://github.com/focalintent/FastLED-Sparkcore/blob/master/firmware/fastpin_arm_stm32.h
+#define MAX_PIN 20
```

Comment:
The downstream repo has `MAX_PIN 19` but then uses pin 20 so I felt it was best to change to 20.